boot/bootdata/hivesys.inf

```diff
@@ -570,6 +570,10 @@ HKLM,"SYSTEM\CurrentControlSet\Control\Keyboard Layouts\0000040c","Layout File",
 HKLM,"SYSTEM\CurrentControlSet\Control\Keyboard Layouts\0000040c","Layout Text",0x00000000,%FRENCH%
 HKLM,"SYSTEM\CurrentControlSet\Control\Keyboard Layouts\0000040c","Layout Display Name",0x00000000,"@%SystemRoot%\system32\input.dll,-5010"
 
+HKLM,"SYSTEM\CurrentControlSet\Control\Keyboard Layouts\0000100c","Layout File",0x00000000,"kbdsf.dll"
+HKLM,"SYSTEM\CurrentControlSet\Control\Keyboard Layouts\0000100c","Layout Text",0x00000000,%SWISS_FRENCH%
+HKLM,"SYSTEM\CurrentControlSet\Control\Keyboard Layouts\0000100c","Layout Display Name",0x00000000,"@%SystemRoot%\system32\input.dll,-5023"
+
```

Comment:
Please put these entries after the canadian_french entries below (lexicographical order)